ovos_plugin_manager/utils/__init__.py

```diff
@@ -179,41 +195,53 @@ def normalize_lang(lang):
 class ReadWriteStream:
     """
     Class used to support writing binary audio data at any pace,
-    optionally chopping when the buffer gets too large
+    with an optional maximum buffer size
     """
 
-    def __init__(self, s=b'', chop_samples=-1):
-        self.buffer = s
+    def __init__(self, s=b'', max_size=None):
```

Comment:
breaking change in init signature

Reply:
https://github.com/OpenVoiceOS/ovos-plugin-manager/pull/256